templates/button.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Default template for Fieldmanager_Button
+ *
+ * @package Fieldmanager
+ */
+?>
+<input
```

Comment:
Something to consider: instead of `input[type=button]`, this could be a `button[type=button]`. If we did that, it would mean that the `button_text` could instead be `button_content` and could theoretically be any html content, like an image (we'd have to piggyback on FM's escaping argument to escape html by default, but allow for an override). I could be convinced either way; I'm curious what others think about it.

Reply:
I refreshed my own memory how it works, and the escaping if we opted for `button` would be really simple:

```php
<button
    type="button"
    ...
>
    <?php echo $this->escape( 'button_content' ); ?>
</button>
```

If an implementor wanted to allow rich text in the field, they'd simply pass something like `[ 'escape' => [ 'button_content' => 'wp_kses_post' ] ]` in their field definition.